packages/odyssey-react-mui/src/Badge.tsx

```diff
@@ -80,17 +81,20 @@ const Badge = ({
       fontFamily: `${odysseyDesignTokens.TypographyFamilyMono}`,
       fontWeight: `${odysseyDesignTokens.TypographyWeightBodyBold}`,
       lineHeight: 1,
-    }),
-    [type, contentIsLongerThanOneChar, odysseyDesignTokens]
-  );
+      transitionDuration: `${odysseyDesignTokens.TransitionDurationMain}`,
+      transitionProperty: `background-color, color`,
+    };
+
+    const shouldHideBadge = badgeContent <= 0 || !badgeContent;
 
-  const shouldHideBadge = badgeContent <= 0 || !badgeContent;
+    if (shouldHideBadge) {
+      return null;
+    }
 
-  if (shouldHideBadge) {
-    return null;
-  }
+    return <Box sx={badgeStyles}>{formattedContent}</Box>;
+  }, [badgeContent, badgeContentMax, type]);
```

Comment:
You need to include `odysseyDesignTokens` in the list of variables for the callback